projects/ngrx.io/content/guide/component-store/write.md

```diff
@@ -89,3 +90,42 @@ export class MoviesPageComponent {
   }
 }
 </code-example>
+
+## `patchState` method
+
+The `patchState` method can be called by providing a partial state object or a partial updater callback.
+
+When the partial state is provided it patches the state with the provided value.
+
+When the partial updater is provided it patches the state with the value returned from the callback.
+
+<code-example header="movies-page.component.ts">
+interface MoviesState {
+  movies: Movie[];
+  selectedMovieId: string | null;
+}
+
+@Component({
+  template: `...`,
+  providers: [ComponentStore],
+})
+export class MoviesPageComponent {
```

Comment:
```suggestion
export class MoviesPageComponent implements OnInit {
```

This is also missing in the `setState` example 🤦‍♂️

Reply:
Good catch, thanks Tim 👍 It's fixed now